spec/controllers/adoptions_controller_spec.rb

```diff
@@ -0,0 +1,363 @@
+require 'spec_helper'
+
+describe Api::V1::AdoptionsController do
+  
+  let(:user) { create(:user, account_active: 'true', permissions: 'adopters_edit') }
+  let(:user2) { create(:user, account_active: 'true', permissions: 'default_user') }
+  let(:user3) { create(:user, account_active: 'true', permissions: 'animals_edit') }
+  let(:user4) { create(:user, account_active: 'true', permissions: 'super_user') }
+
+  let(:adopter) { create(:adopter) }
+  let(:adopter2) { create(:adopter, blacklisted: 'true') }
+  let(:adopter3) { create(:adopter) }
+  let(:species) { create(:species, name: 'Perro', adoptable: 'true') }
+  let(:species2) { create(:species, name: 'Gallina', adoptable: 'false') }
+  let(:animal) { create(:animal, species_id: species.id, vaccines: 'true', castrated: 'true', type: 'Adoptable') }
+  let(:animal1) { create(:animal, species_id: species.id, vaccines: 'false', castrated: 'true', type: 'Adoptable') }
+  let(:animal2) { create(:animal, species_id: species.id, vaccines: 'true', castrated: 'false', type: 'Adoptable') }
+  let(:animal3) { create(:animal, species_id: species2.id, vaccines: 'true', castrated: 'true') }
+
+  let(:adoption) { create(:adoption, adopter_id: adopter3.id, animal_id: animal.id, date: '2016-11-01') }
+
+  let!(:params) do
+    {
+      adopter_id: adopter.id,
+      animal_id:  animal.id,
+      date: '2016-11-01'
+    }
+  end
+
+  describe 'POST #create' do
+    context 'exitoso' do
+      context 'por un usuario con permiso para editar adoptantes' do
+        before(:each) do
+          request.headers['X-USER-TOKEN'] = user.authentication_token
+        end
+
+        it 'devuelve codigo 201' do
+          post :create, adoption: params, format: 'json'
```

Comment:
No encuentro forma de mover esa call  al bloque `before(:each)` y poder ejecutar este example: `expect { post :create, adoption: params, format: 'json' }.to change { Adoption.count }.by(+1)`

Lo mismo sucede para los demás comentarios que me hiciste sobre esto.
Lo hablé con Berna y el tampoco conoce alguna forma. 

Si conoces alguna forma de skipearlo, avisame, sino lo pasamos asi subo el spec de comments y cerramos este.